src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtSentencingService.kt

```diff
@@ -366,20 +370,25 @@ fun Charge.toNomisCourtCharge(): OffenderChargeRequest = OffenderChargeRequest(
   offenceDate = this.offenceStartDate,
   offenceEndDate = this.offenceEndDate,
 // TODO dps has text that 'mainly' matches nomis but there are also non-matching values on T3
-  resultCode1 = this.outcome,
+  resultCode1 = getHardcodedNomisResultCode(this.outcome),
 // TODO do dps provide this?
   mostSeriousFlag = false,
   // TODO determine if this comes from DPS or is it determined
   offencesCount = 1,
 )
 
+fun getHardcodedNomisResultCode(dpsResultCode: String): String {
+  // map to remand or imprisonment NOMIS codes
+  return if ("remand" in dpsResultCode.lowercase()) HARDCODED_REMAND_RESULT_CODE else HARDCODED_IMPRISONMENT_RESULT_CODE
```

Comment:
just to give a slight variation when testing.  Decision on outcome mappings not made yet